package.json

```diff
@@ -91,6 +91,8 @@
 		"@ianvs/prettier-plugin-sort-imports": "4.5.1",
 		"@mui/icons-material": "7.2.0",
 		"@mui/material": "7.2.0",
+		"@sentry/react": "9.43",
```

Comment:
there's a more recent version available but this is strictly tied to the version that's needed by `@sentry/electron`. They even provide a useful type error that tells you which SDK you're supposed to install 😄